wp-content/themes/pub/wporg-learn-2024/patterns/contribute-page-content.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Title: Contribute Page Content
+ * Slug: wporg-learn-2024/contribute-page-content
+ * Inserter: no
+ */
+
+?>
+ 
+<!-- wp:heading {"style":{"typography":{"fontSize":"36px","fontStyle":"normal","fontWeight":"400"}},"fontFamily":"eb-garamond"} -->
+<h2 class="wp-block-heading has-eb-garamond-font-family" style="font-size:36px;font-style:normal;font-weight:400">
+	<?php esc_html_e( 'Contribute', 'wporg-learn' ); ?>
+</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"},"spacing":{"margin":{"bottom":"40px","top":"0px"}}},"fontSize":"normal","fontFamily":"inter"} -->
+<p class="has-inter-font-family has-normal-font-size" style="margin-top:0px;margin-bottom:40px;font-style:normal;font-weight:400">
+	<?php esc_html_e( "Here's how you can get involved and create content for Learn WordPress", 'wporg-learn' ); ?>
+</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"600"}},"fontSize":"extra-large","fontFamily":"inter"} -->
+<h2 class="wp-block-heading has-inter-font-family has-extra-large-font-size" style="font-style:normal;font-weight:600">
+	<?php esc_html_e( 'Create a Tutorial', 'wporg-learn' ); ?>
+</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"},"spacing":{"margin":{"bottom":"var:preset|spacing|20","top":"var:preset|spacing|20"}}},"fontSize":"small","fontFamily":"inter"} -->
+<p class="has-inter-font-family has-small-font-size" style="margin-top:var(--wp--preset--spacing--20);margin-bottom:var(--wp--preset--spacing--20);font-style:normal;font-weight:400">
+	<?php esc_html_e( 'Tutorials can be on any topic related to WordPress, can be for any level of experience, and in any language. To get started, check out this ', 'wporg-learn' ); ?>
+	<a href="https://learn.wordpress.org/workshop/how-to-submit-a-workshop/"><?php esc_html_e( 'tutorial about creating tutorials', 'wporg-learn' ); ?></a><?php esc_html_e( '. Then ', 'wporg-learn' ); ?>
+	<a href="https://learn.wordpress.org/workshop-presenter-application/"><?php esc_html_e( 'submit your tutorial presenter application', 'wporg-learn' ); ?></a><?php esc_html_e( '.', 'wporg-learn' ); ?>
+</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:embed {"url":"https://videopress.com/v/52hW5V7s","type":"video","providerNameSlug":"videopress","responsive":true,"className":"wp-embed-aspect-16-9 wp-has-aspect-ratio"} -->
+<figure class="wp-block-embed is-type-video is-provider-videopress wp-block-embed-videopress wp-embed-aspect-16-9 wp-has-aspect-ratio"><div class="wp-block-embed__wrapper">
+https://videopress.com/v/52hW5V7s
+</div></figure>
+<!-- /wp:embed -->
+
```

Comment:
I think we may want to remove this section, given Tutorials are being deprecated. What do you think @jonathanbossenger @bsanevans ?

Reply:
lesson plans as well, or do we still encourage people to contribute lesson plans?